src/transformation/convex_hull3/validation.rs

```diff
@@ -28,10 +29,10 @@ pub fn check_facet_links(ifacet: usize, facets: &[TriangleFacet]) {
 }
 
 /// Checks if a convex-hull is properly formed.
-#[cfg(feature = "std")]
-pub fn check_convex_hull(points: &[Point3<Real>], triangles: &[[u32; 3]]) {
-    use std::println;
-
+pub fn check_convex_hull(
+    points: &[Point3<Real>],
+    triangles: &[[u32; 3]],
+) -> Option<ConvexHullError> {
```

Comment:
Returning a `Result` would communicate better the intent.

Reply:
Fixed.